testnet/stacks-node/src/syncctl.rs

```diff
@@ -382,34 +426,40 @@ impl PoxSyncWatchdog {
                 get_epoch_time_secs() + self.steady_state_burnchain_sync_interval;
         }
 
+        // inform any listeners about the new burnchain height
+        self.relayer_comms
+            .set_burnchain_tip_height(burnchain_height);
+
+        let ibbd = PoxSyncWatchdog::infer_initial_burnchain_block_download(
+            burnchain,
+            burnchain_tip.block_snapshot.block_height,
+            burnchain_height,
+        );
+
         // unconditionally download the first reward cycle
         if burnchain_tip.block_snapshot.block_height
             < burnchain.first_block_height + (burnchain.pox_constants.reward_cycle_length as u64)
         {
             debug!("PoX watchdog in first reward cycle -- sync immediately");
             sleep_ms(10_000);
```

Comment:
Do we want to revisit this 10s sleep?

Reply:
I also recall we had added a 30s sleep somewhere just after the mainnet launch, do we still need that?

Reply:
Also, should we turn it into a constant / setting?

Reply:
This is here mainly to prevent the node from eating 100% CPU (and slamming the Bitcoin node) while processing the first reward cycle.  Can change to the sync constant `self.steady_state_burnchain_sync_interval`.

Reply:
Addressed in 08db53b